holmes/plugins/toolsets/bash/bash_toolset.py

```
@@ -233,35 +299,79 @@ def get_parameterized_one_liner(self, params: Dict[str, Any]) -> str:
+            cli_prefixes = get_cli_approved_prefixes()
+            if cli_prefixes and self.config:
+                # Merge without duplicates
+                existing = set(self.config.allow)
+                for prefix in cli_prefixes:
+                    if prefix not in existing:
+                        self.config.allow.append(prefix)
+                logging.debug(f"Merged {len(cli_prefixes)} CLI-approved prefixes")
+        except ImportError:
+            # interactive module may not be available in all contexts
+            pass
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)